### PR TITLE
Remove the emulate command

### PR DESCRIPTION
**Pull request by etabakov**

### CODE REVIEW

**Comment by etabakov** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

File: lib/commands/emulate.ts [hunk 1]
```diff
@@ -1,68 +0,0 @@
...
-			availableDevices: this.$options.availableDevices,
-			platformTemplate: this.$options.platformTemplate,
-			provision: this.$options.provision,
-			teamId: this.$options.teamId,
-			keyStoreAlias: this.$options.keyStoreAlias,
-			keyStoreAliasPassword: this.$options.keyStoreAliasPassword,
-			keyStorePassword: this.$options.keyStorePassword,
-			keyStorePath: this.$options.keyStorePath
-		};
-		return this.$platformService.emulatePlatform(args[0], appFilesUpdaterOptions, emulateOptions, this.$projectData, this.$options);
```
Comment (rosen-vladimirov) [inline lib/commands/emulate.ts]: the `emulatePlatform` method of `platformService` is not used by anyone else, shouldn't we delete it as well?
Also I believe we no longer need `IEmulatePlatformOptions` interface

Comment (etabakov) [inline lib/commands/emulate.ts]: Thanks, removed those as well.

**Pull request merged**